src/test/java/com/amazon/opendistroforelasticsearch/sql/esintgtest/QueryIT.java

```diff
@@ -1671,6 +1671,41 @@ public void fieldCollapsingTest() throws IOException {
         Assert.assertEquals(21, hits.length());
     }
 
+    @Test
+    public void backticksQuotedIndexNameTest() throws Exception {
+        AdminClient adminClient = this.admin();
+        TestUtils.createTestIndex(adminClient, "bank_unquote", "bank_unquote", null);
+        TestUtils.loadBulk(ESIntegTestCase.client(),
+                "/src/test/resources/bank_for_unquote_test.json", "bank");
+
+        final String query = "SELECT lastname FROM `bank` ORDER BY age LIMIT 3";
+        JSONObject response = executeQuery(query);
+        JSONArray hits = getHits(response);
+
+        assertThat("bank", equalTo(((JSONObject) hits.get(0)).query("/_index")));
+    }
+
+    @Test
+    public void backticksQuotedFieldNamesTest() {
```

Comment:
Please add one more case for field alias and assert name in JDBC response is unquoted too, ex. `` SELECT `b`.`lastname` AS `name` ... ``